src/services/ws.ts

```diff
@@ -213,30 +237,30 @@ function sendTransactionEntityChangesToAllClients() {
 }
 
 function syncPullInProgress() {
-    sendMessageToAllClients({ type: 'sync-pull-in-progress', lastSyncedPush });
+    sendMessageToAllClients({ type: 'sync-pull-in-progress', lastSyncedPush: lastSyncedPush || undefined });
```

Comment:
Why do we need to send undefined instead of null now? What has changed (beyond the type for lastSyncedPush in Message) to require this extra work?

Reply:
@j-tirelli , it seems I have an unconcious preferences to avoid `T | null` and use the `?` suffix to variables instead. This forced me to force `null` to `undefined`.

But I agree it's much cleaner not to do it.